elk/lightcurve.py

```diff
@@ -613,14 +616,15 @@ def diagnose_lc_periodogram(self, output_path, freq_bins='auto', identifier='',
         aperture_powers = np.asarray(self.pixel_periodograms)[self.star_mask.flatten()]
         assert len(aperture_powers) > 0, "No pixel periodograms found - did you run `self.correct_lc`?"
 
-        if save_simbad_queries:
-            simbad_queries = {'ra': [], 'dec': [], 'freq_lower': [], 'freq_upper': []}
```

Comment:
Curious: Why do you only want to record the peak frequency? I feel like having some sort of uncertainty range with the lower/upper values could be useful for checking the list of stars that you return?

Reply:
That's true. We can do that as well, but I want to query on the peak itself, and have it accessable in the returned table, which star is in the query for which peak.

Reply:
Okay gotcha yeah I think both makes sense - I was imagining a situation where your periodogram may not be evaluating at the precise frequency since the array can only be so wrong, so if you use the peak alone you may miss things that are close but exactly at the peak?